docs/docs/create-apps/general-app.md

```diff
@@ -14,50 +17,105 @@ Before deploying/launching your app with JHub Apps Launcher, you'll need:
+created. 
+
+![Create app form, thumbnail section](/img/create_app_thumbnail.png)
+
+After submitting this form, users are redirected to the second stage of the process - server
+selection. Here, users must select the server/instance type (i.e. machines with CPU/RAM/GPU 
+resources) in which to deploy the app. This is only applicable if you're using 
+`KubeSpawner` as the spawner for JupyterHub.
+
+![Create app form, select server page](/img/create_app_select_server.png)
+
+Once you click "Deploy App", you'll be redirected to the app page which will automatically refresh
+when the app is ready for viewing. 
+
+### Details about each field are included below:
+
+| Field                                                           | Description  |
+| --------------------------------------------------------------- | ------------ |
+| Name                                                           | Provide a meaningful name for your application. |
+| Description (optional)                                         | Add addition information about the application. |
+| Framework                                                      | Select the framework used by your application from the dropdown. |
+| Software Environment                                           | Provide the environment used while developing your notebook/script which has `jhsingle-native-proxy`, the corresponding framework (e.g., `panel`, `gradio`, etc.), and additional requirements documented in framework-specific pages. |
+| Filepath                                                       | Provide the path (from root in JupyterLab) to your application's code. |
+| Keep alive                                                     | Toggle on to keep app alive indefinitely. If toggled off, app server will be shut down according to the `jupyter-idle-culler` timeout settings. |
+| Environment Variables (optional)                               | The set of environment variables, defined as valid JSON, to set before starting the app. This is useful when creating custom apps. |
+| Custom Command (required if the Framework is "Custom Command") | Python command to start an arbitrary app (only visible if using "Custom Command" option). |
+| Sharing                                                        | To share the application with a set of users and or groups. |
+| Allow Public Access                                            | Toggle to share the application with anyone on the internet, including unauthenticated users. |
+| Thumbnail (optional)                                           | Choose a meaningful thumbnail for your application. The default thumbnail is the application framework's logo. |
+
 
-1. From the JHun App Launcher Home Screen, click on the **"Create App"** button in the top-right corner.
+## Modifying existing apps
 
-2. Fill in app-creation form with the details below.
+Once your app is running, it will appear on the jhub-apps homepage as an individual card.
```

Comment:
JHub Apps homepage

I think we have been using `jhub-apps` when talking about the library and `JHub Apps` for the UI/dashboard(?)